### PR TITLE
Reorganizes CollabswarmDocument.

### PR DESCRIPTION
**Pull request by robotoer**
Added some more comments and moved helpers into one area

### CODE REVIEW

File: packages/collabswarm/src/collabswarm-document.ts [hunk 1]
```diff
@@ -436,6 +410,16 @@ export class CollabswarmDocument<
...
 
+  /**
+   * Subscribes a change handler to the document. Use this method to receive real-time
+   * updates to the document.
+   *
+   * @param id A unique id for this handler. Used to unsubscribe this handler.
+   * @param handler A function that is called when a change is received.
+   * @param originFilter Determines what kinds of change events trigger the handler.
+   *     'remote' indicates that the change was received from a remote peer.
+   *     'local' indicates that the change was received from the local document.
```
Comment (robotoer) [inline packages/collabswarm/src/collabswarm-document.ts:421]:
```suggestion
   *     'local' indicates that the change was received from the local document.
   *     'all' indicates that all changes should be handled.
```

File: packages/collabswarm/src/collabswarm-document.ts [hunk 2]
```diff
@@ -507,4 +497,34 @@ export class CollabswarmDocument<
     // Fire change handlers.
     this._fireLocalUpdateHandlers([hash]);
   }
+
+  public async pin() {
+    // Apply local change w/ automerge.
```
Comment (lifekaizen) [inline packages/collabswarm/src/collabswarm-document.ts:502]: `// Apply local change w/ CRDT provider.`

**Review by lifekaizen — APPROVED** [review]

File: packages/collabswarm/src/collabswarm-document.ts (hunk 2 above)
Comment (lifekaizen) [reply packages/collabswarm/src/collabswarm-document.ts:502]: Did you mean to specify Automerge?

Comment (robotoer) [reply packages/collabswarm/src/collabswarm-document.ts:502]: Ah whoops, good catch

Comment (robotoer) [reply packages/collabswarm/src/collabswarm-document.ts:502]:
```suggestion
    // Apply local change w/ CRDT provider.
```

**Pull request merged**